chromium/background.js

```diff
@@ -227,14 +227,11 @@ function onBeforeRequest(details) {
   const uri = new URL(details.url);
 
   // Should the request be canceled?
-  var shouldCancel = (
-    httpNowhereOn &&
-    uri.protocol === 'http:' &&
-    !/\.onion$/.test(uri.hostname) &&
-    !/^localhost$/.test(uri.hostname) &&
-    !/^127(\.[0-9]{1,3}){3}$/.test(uri.hostname) &&
-    !/^0\.0\.0\.0$/.test(uri.hostname)
```

Comment:
@Hainish I thought you mean I removed the entire statement. Sorry. `0.0.0.0` is not a valid IPv4 address. Not sure why we have it here.